### ISSUE TITLE
Game crashes when attempting to play a simfile

### ISSUE THREAD

**Issue by kittengamer5023** [DarkBahamut162/itg3encore#5]
I have tested this several times and it is definitely a theme issue.

I have tried using both Outfox (Alpha V) and ITGmania (0.6.0) and only while using the ITG3 Encore theme the game has crashed when attempting to open the attached simfile, and only that specific simfile. I tried saving the file again as an SSC (it was a SM file before) and the only difference was that the game failed to load it rather than crashing, as it immediately displayed the Song Cleared text after the chart would have been loaded.

[Cross Over.zip](https://github.com/DarkBahamut162/itg3encore/files/11770226/Cross.Over.zip)

**Comment by DarkBahamut162** [comment]
This is a known issue. And I might actually need to fix this:

For quicker transitions between SelectMusic/StageInformation and Gameplay, I'm checking the BGChanges on the simfile itself and start everything off from the earliest point from there (to not play the video later than intended and doing potential other glitches).

I had the same problems on PIU simfiles having black videos at beat -10000.
ITGMania (and maybe SM itself) seem to have a cap from which earliest beat the game should start off on, as it gives me the following crash:

`SongPosition.cpp:26 Song beat -5000.026855 at -1500.009033 seconds is less than -2000!`

And by the looks of it, this simfile has something similar. On what I have calculated, the song would start off 25 minutes earlier than intended, even if it would have went in successfully. I should actually do some sort of sanity check on the BGChanges and only pass over timings if they are within 20 second from the start of the song.

Also: ITGMania is currently on 0.6.1 but this issue is still there.
BUT I've checked the same simfile on OutFox (0.4.19 & 0.5.0-041) and there it works, but with the problem explained above.
